terraform/eks/daemon/default_resources/default_amazon_cloudwatch_agent.json

```diff
@@ -2,7 +2,8 @@
   "logs": {
     "metrics_collected": {
       "kubernetes": {
-        "metrics_collection_interval": 30
+        "metrics_collection_interval": 30,
+        "enhanced_container_insights": true
```

Comment:
Not going to block -- would it be better to have a separate test for enhanced_container_insights ?

Reply:
Will be addressed in a separate test which tests every observation